modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase

```diff
@@ -407,7 +407,7 @@ definition {
 				fieldName = "customObjectField",
 				fieldType = "String",
 				isRequired = "false",
-				objectName = "CustomObject161922");	
+				objectName = "CustomObject161922");
```

Comment:
The changes in e4ddc8ba1c2eda83faca5af287ce5e99790994f9 should be made in the commits that caused them and not in a separate commit like this. Essentially, you're making a mistake in a commit and using another commit to fix it. It's better to go back and fix the original commit so that you never made the mistake in the first place.

Reply:
@JhossephG let me try explaining this in another way. This commit, 25de24effa51100052d0f58150b2f5b2569d6127, shouldn't exist. All of the changes in this commit are to fix mistakes made in previous commits. You should go back and fix the original commit instead of using a new commit.

Reply:
Don't just fix this error seen in the comment, please go and fix all of the errors in this commit in the original commits.